roles/default/kiali-deploy/templates/openshift/role.yaml

```diff
@@ -97,4 +97,14 @@ rules:
   - tokenreviews
   verbs:
   - create
+- apiGroups: [""]
+  resourceNames:
+  - cacerts
+  - istio-ca-secret
+  resources:
+  - secrets
+  verbs:
+  - create
+  - list
+  - watch
```

Comment:
Not sure why this is here. Is the Kiali Server actually creating these secrets?

Reply:
Note - there isn't any analogous change to the server helm-chart template - I assume this needs to be reverted. I don't think this should be here.